hack/local-monitor-testing/startMDMNetwork.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -e
+set +x
+
+BASE=$( git rev-parse --show-toplevel)
+SOCKETFILE="$BASE/cmd/aro/mdm_statsd.socket"
+
+echo "Using:"
+
+echo "Resourcegroup = $RESOURCEGROUP"
+echo "User          = $USER"
+
+VMName="$USER-mdm-link"
+CLOUDUSER="cloud-user"
+
+
+PUBLICIP=$( az vm list-ip-addresses --name $VMName -g $RESOURCEGROUP | jq -r '.[0].virtualMachine.network.publicIpAddresses[0].ipAddress' )
+
+echo "Found IP $PUBLICIP, starting socat on the mdm-link vm"
+ssh $CLOUDUSER@$PUBLICIP "sudo socat -v TCP-LISTEN:12345,fork UNIX-CONNECT:/var/etw/mdm_statsd.socket" &
```

Comment:
Nit: dangling forked processes in the script. Could use information what is left running in the background and how to stop it.